src/screens/IndexScreen.js

```diff
@@ -616,9 +653,10 @@ export const IndexScreen = ({ navigation, route }) => {
           }}
         </Query>
       )}
-      {query === QUERY_TYPES.POINTS_OF_INTEREST && (
-        <IndexMapSwitch filter={filterType} setFilter={setFilterType} />
-      )}
+      {query === QUERY_TYPES.POINTS_OF_INTEREST &&
+        filterType.find((i) => i.selected && i.title === 'Listenansicht') && (
```

Comment:
please do not use `i` as it suggests some `index` counter. 
earlier in this file you have used `filterType.find((entry) => entry.selected);`. this would be the better alternative here as well.

Reply:
done with #921